embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParser.java

```diff
@@ -0,0 +1,112 @@
+package org.embulk.spi.time;
+
+import org.jruby.runtime.ThreadContext;
+
+import java.util.HashMap;
+import java.util.List;
+
+import static org.embulk.spi.time.StrptimeParser.FormatBag.has;
+import static org.jruby.RubyRational.newRationalCanonicalize;
+
+/**
+ * This class has {@code StrptimeParser} and provides methods that are calls from JRuby.
+ */
+public class RubyDateParser
```

Comment:
We still need this `RubyDateParser` because this is required for executing `test/mri/date/test_date_strptime.rb` with new `StrptimeParser`.